wcomponents-theme/src/main/js/wc/ui/containerload.js

```diff
@@ -166,10 +167,10 @@ function(shed, triggerManager, ajaxRegion, initialise, uid, Widget, classList, c
 		function init() {
 			if (!inited) {
 				inited = true;
-				shed.subscribe(shed.actions.EXPAND, instance.onexpand);
-				shed.subscribe(shed.actions.COLLAPSE, handleCollapseOrHide);
-				shed.subscribe(shed.actions.SHOW, instance.onshow);
-				shed.subscribe(shed.actions.HIDE, handleCollapseOrHide);
+				event.add(document.body, shed.events.EXPAND, instance.onexpandlistener);
```

Comment:
We aim for camelCase close to 100%. Not sure why it wasn't adhered to before and I can see you have simply carried that over. Unless I am missing something I suggest we make it camelCase now.